src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -729,6 +729,7 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
                     interactive={!isReadOnly}
                     brickRoadIndicator={merchantError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : ''}
                     error={merchantError ? translate('common.error.fieldRequired') : ''}
+                    rightLabel={canUseViolations && PolicyUtils.isPaidGroupPolicy(policy) ? translate('common.required') : ''}
```

Comment:
Unsure if the `PolicyUtils.isPaidGroupPolicy(policy)` is redundant. `isMerchantRequired` checks for if the policy is of type `CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT` which might be a level higher than paid group policy. Is that the case @cead22 ?

Reply:
I think so, can you test to confirm? We can get rid of `canUseViolations` here as well